sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Added support for OAuth copy sources for synchronous copy operations.
 - Added support for Parquet as an input format in BlockBlobClient.Query().
 - Added optimization to unwrap encryption key once for DownloadTo and OpenRead when Client Side Encryption is enabled.
+- Added support for RequestConditions parameter validation.  If a request condition is set for an API that doesn't support it, and Arguement Exception will be thrown.
```

Comment:
we should mention compat switch in changelog.

Reply:
compat switch?